File: Level-0/98.java
class Solution {
    public int[] solution(int num, int total) {
        int[] answer = new int[num];
        int n = 0;
        for(int i = 1; i < num; i++){
            n += i;
        }
        
        int start = (total - n) / num;
        for(int i = 0; i < num; i++){
            answer[i] = start + i;
        }
        
        return answer;
    }
}

File: Level-0/98.js
function solution(num, total) {
    let answer = [];
    const value = Math.ceil(total / num);
    const idx = Math.floor(num / 2);
    let first = value - idx;
    for(let i = 0; i < num; i++) {
        answer.push(first++);
    }
    return answer;
}
